.github/workflows/python-checks.yml

```diff
@@ -57,7 +57,13 @@ jobs:
       - name: s3torchconnectorclient unit tests
         run: pytest s3torchconnectorclient/python/tst/unit --hypothesis-profile ci --hypothesis-show-statistics
       - name: s3torchconnector unit tests
-        run: pytest s3torchconnector/tst/unit --hypothesis-profile ci --hypothesis-show-statistics
+        run: pytest s3torchconnector/tst/unit --ignore s3torchconnector/tst/unit/lightning --hypothesis-profile ci --hypothesis-show-statistics
+
+      - name: Install Lightning dependency
+        run: |
+          python -m pip install -e "s3torchconnector[lightning]"
```

Comment:
Latest Lightning version is 2.1.4. Should we try to have a matrix here with lightning versions that we specifically install and run tests on?

Reply:
I'm not sure, though I think if we do decide to do this, we should do it in a separate PR, possibly along with a matrix of pytorch versions